vllm/engine/arg_utils.py

```diff
@@ -851,6 +864,29 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
         vllm_group.add_argument("--additional-config",
                                 **vllm_kwargs["additional_config"])
 
+        # Signature verification arguments
+        sv_kwargs = get_kwargs(SignatureVerificationConfig)
+        sv_group = parser.add_argument_group(
+            title="SignatureVerificationConfig",
+            description=SignatureVerificationConfig.__doc__,
+        )
+        sv_group.add_argument("--verification-method",
+                              choices=[f.value for f in VerificationMethod],
```

Comment:
If you changed `VerificationMethod` to be a `Literal` then this special case would not be necessary, `get_kwargs` would automatically populate `choices` with the contents of the `Literal`

Reply:
Done.